CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
+
+### Added
+- `ui/Cell` prop `grow` to expand its size to the container
```

Comment:
If you are making a change of ui package, please add a change log under packages/ui/CHANGELOG.md

Reply:
Also, the module name would be `ui/Layout.Cell`.

Reply:
OK. I will fix it.